libs/core-ui/src/lib/Interfaces/IDataset.ts

```diff
@@ -11,6 +13,7 @@ export interface IDataset {
   categorical_features: string[];
   class_names?: string[];
   target_column?: string;
+  data_balance_measures?: IDataBalanceMeasures;
```

Comment:
More of a conceptual question: Are these only going to be shown for the dataset as a whole? What if we have a global cohort that shows only a subset of the data? Does it show the data balance measures for that subset, or still globally?
From the adultCensus.ts file I'm gathering that these are all things that are precomputed, which would indicate that we don't show this per-cohort, but rather for the whole dataset. I do feel like this would totally be possible and perhaps be a more natural fit, but then I think that's more of a PM/design question and should go to the people who make such decisions as they probably have thought about it more than me 🙂

Reply:
You are right: the measures are currently precomputed using the whole dataset for whatever columns user specifies in `columns_of_interest`!

I think it could also be possible to compute per-cohort, provided that the dataset isn't so big that it slows things down. You saw from the Python module that the computation logic is fairly simple -- lots of `groupby`s to compute `P(feature), P(feature and positive label), P(positive label)` etc. (atleast for feature balance measures)!

I think these per-dataset insights are are the first step to something concrete, and I can totally see it working per-cohort like the rest of the dashboard.

Reply:
Good point! Tagging @memoryz, @imatiach-msft, and my PM @martthalch for your guys' thoughts 😊

Reply:
I agree with all that was said above. I think we'll focus on just supporting all data since there is a lot more to consider when adding cohort support from what I understand. And yah, making sure that its clear that cohorts aren't supported, maybe with some UI updates to provide context or something like that

Reply:
+1 I agree with the thoughts above.  I would suggest to get this working for all data first before trying to do it for all cohorts - and we also may indeed want to limit this in some scenarios, like big data, to only compute over all data and not at cohort level in the future.